### PR TITLE
Updating the OTel instrumentation package versions to 1.8.0

### PR DESCRIPTION
**Pull request by samsp-msft**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/aspire/pull/3488)

### CODE REVIEW

File: Directory.Packages.props [hunk 1]
```diff
@@ -123,9 +123,9 @@
     <PackageVersion Include="OpenTelemetry.Exporter.OpenTelemetryProtocol" Version="1.8.0" />
     <PackageVersion Include="OpenTelemetry.Extensions.Hosting" Version="1.8.0" />
     <PackageVersion Include="OpenTelemetry.Instrumentation.GrpcNetClient" Version="1.7.0-beta.1" />
```
Comment (eerhardt) [inline Directory.Packages.props:125]:
```suggestion
    <PackageVersion Include="OpenTelemetry.Instrumentation.GrpcNetClient" Version="1.8.0-beta.1" />
```

Might as well change this too.

**Review by danmoseley — APPROVED** [review]

**Review by eerhardt — APPROVED** [review]

**Comment by samsp-msft** [comment]
@eerhardt - that change is now breaking Orleans...

**Comment by eerhardt** [comment]
It just needs to be mirrored. I kicked it off. Should be done soon.

**Pull request merged**